### PR TITLE
Update project version to 4.12

### PR DESCRIPTION
**Pull request by clacroix12**
Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

**Review by vavuthu — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="ocs-ci",
-    version="4.6.0",
+    version="4.12.0",
```
Comment (vavuthu) [inline setup.py:12]: isn't it 4.13 ?

Comment (vavuthu) [inline setup.py:12]: isn't it 4.13 ?

Comment (petr-balogh) [reply setup.py:12]: No, we want to have this in release 4.12 branch. 4.13 we will put in your pr to default to 4.13 Vijay

**Review by petr-balogh — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (vavuthu) [reply setup.py:12]: > No, we want to have this in release 4.12 branch. 4.13 we will put in your pr to default to 4.13 Vijay

got it

**Pull request merged**